app/models/search.rb

```diff
@@ -1,6 +1,8 @@
 require 'api_entity'
 
 class Search
+  EUROPEAN_COUNTRIES = %w[AT BE BG CY CZ DE DK EE ES FI FR GR HR HU IE IT LT LU LV MT NL PL PT RO SE SI SK]
```

Comment:
Linters complaining about this line - I believe general policy is linters not enforced because of historical code but anything new should pass the linter

Reply:
I agree, here is it missing the `.freeze`, I'll fix it